### PR TITLE
Remove instance puts staging

### PR DESCRIPTION
**Pull request by fppt**

### CODE REVIEW

File: mindmaps-graph/src/main/java/io/mindmaps/example/MovieGraphFactory.java [hunk 1]
```diff
@@ -309,8 +280,10 @@ private static void buildRules() {
                 .addConclusion(person).addConclusion(genre).addHypothesis(hasCast);
     }
 
-    private static Entity putEntity(EntityType type, String name) {
-        return mindmapsGraph.putEntity(name.replaceAll(" ", "-").replaceAll("\\.", ""), type);
+    private static Entity putEntityWithName(EntityType type, String value) {
```
Comment (aelred) [inline mindmaps-graph/src/main/java/io/mindmaps/example/MovieGraphFactory.java]: Pedantry alert: this should be "addEntityWithName"

Comment (fppt) [inline mindmaps-graph/src/main/java/io/mindmaps/example/MovieGraphFactory.java]: I will fix that in my next PR. Don't want to run all the tests over again just for one internal refactor.

**Comment by aelred** [comment]
Actually, I agree with @bolerio . The original plan was to deprecate the methods before the 0.1.0 release. Given that 0.1.0 is out, we don't have that pressure anymore and we can do this properly.

I also think deprecation might be premature. We don't currently provide useful or convenient ways to have unique resources, so deprecating these methods will leave users having to do annoying work-arounds, or lots of manual boilerplate stuff.

In fact, we've already started doing annoying work-arounds in our tests: https://github.com/mindmapsdb/mindmapsdb/pull/232/files#diff-d6c61ac842d51d6047a62aeb491c224aR85).

If we merge this, I can imagine a situation where we have to rush to provide a (poorly-designed) alternative, or we will end up just reverting the change entirely and un-deprecate those methods. Neither is ideal!

**Comment by fppt** [comment]
I will hold off on merging this PR. Primarily because I did feel the pain with having to work around not having the puts anymore. It leads to a lot of code which used to be just one simple line. @haikalpribadi @bolerio Since we have a design meeting later today it might be worth chatting about this issue.

**Comment by fppt** [comment]
@haikalpribadi @bolerio Hey guys, If we want this change in by next release (i.e. end of the month) then we need to make a decision asap. This is a big change to start pushing.

**Pull request closed without merging**